scipy/stats/_distn_infrastructure.py

```diff
@@ -1587,11 +1587,12 @@ def pdf(self, x, *args, **kwds):
         args, loc, scale = self._parse_args(*args, **kwds)
         x, loc, scale = map(asarray, (x, loc, scale))
         args = tuple(map(asarray, args))
-        x = asarray((x-loc)*1.0/scale)
+        dtyp = np.find_common_type([x.dtype, np.float_], [])
```

Comment:
``np.float`` will be deprecated, there was a thread about why it's bad to use it on numpy-discussion recently.

http://docs.scipy.org/doc/numpy/user/basics.types.html says: ``float_ : Shorthand for float64.`` 

Python ``float`` is also 64-bit and IIRC under the hood a C double.